pkg/userdata/bottlerocket/prepare.go

```diff
@@ -1,14 +1,36 @@
 package bottlerocket
 
 import (
+	"path/filepath"
+
 	"github.com/go-logr/logr"
 	"github.com/mrajashree/etcdadm-bootstrap-provider/pkg/userdata"
+	"sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1alpha3"
+)
+
+const (
+	orgCertsPath = "/etc/etcd/pki"
+	newCertsPath = "/var/lib/etcd/pki"
 )
 
 func prepare(input *userdata.BaseUserData) {
 	input.Header = cloudConfigHeader
 	input.WriteFiles = append(input.WriteFiles, input.AdditionalFiles...)
 	input.SentinelFileCommand = sentinelFileCommand
+	patchCertPaths(input)
+}
+
+func patchCertPaths(input *userdata.BaseUserData) {
+	// hacky. new array bc I need to update the original object and these are not pointers
+	files := make([]v1alpha3.File, 0, len(input.WriteFiles))
+	for _, file := range input.WriteFiles {
+		if filepath.Dir(file.Path) == orgCertsPath {
+			file.Path = filepath.Join(newCertsPath, filepath.Base(file.Path))
+		}
+		files = append(files, file)
+	}
```

Comment:
I think we can still update using an index while ranging over input.WriteFiles? this way we won't need new slice
```
	for ind, file := range input.WriteFiles {
		if filepath.Dir(file.Path) == orgCertsPath {
			file.Path = filepath.Join(newCertsPath, filepath.Base(file.Path))
		}
		files[ind] = file
	}
```